english/transform/transform.md

```diff
@@ -5,4 +5,9 @@ tower over you. Clearly, you have shrunk.
 You feel... spongy. Squishy, even. You try to hold up your hands, but you don't have any! Looking down, you see the
 golden ring around your waist like a belt. The rest of you is white, soft, and cylindrical.
 
-You have turned into a marshmallow! The fear of chocolate and graham crackers fills you. 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.